### PR TITLE
Update sulu

### PR DESCRIPTION
**Pull request by luca-rath**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | #issuenum
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

Update sulu to 2.2

### CODE REVIEW

**Comment by nnatter** [comment]
Think this looks good, but I would like to wait for https://github.com/sulu/skeleton/pull/75 before merging this 🙂

File: config/packages/fos_http_cache.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+fos_http_cache:
+    proxy_client:
+        symfony:
+            use_kernel_dispatcher: true
```
Comment (nnatter) [inline config/packages/fos_http_cache.yaml:4]: think i would remove this similar to whats done in https://github.com/sulu/skeleton/pull/75/files

Comment (luca-rath) [reply config/packages/fos_http_cache.yaml:4]: This file is still there in the skeleton ... I removed that config from `config/packages/prod/sulu_http_cache.yaml`

Comment (nnatter) [reply config/packages/fos_http_cache.yaml:4]: oh you are right 🙂

**Pull request merged**